ocs_ci/ocs/utils.py

```diff
@@ -986,6 +989,30 @@ def run_must_gather(log_dir_path, image, command=None, cluster_config=None):
     return mg_output
 
 
+def collect_ceph_external(path):
+    """
+    Collect ceph commands via cli tool on External mode cluster
+
+    Args:
+        path(str): The destination for saving the ceph files [output ceph commands]
+
+    """
+    try:
+        kubeconfig_path = os.path.join(
+            config.ENV_DATA["cluster_path"], config.RUN["kubeconfig_location"]
+        )
+        current_dir = Path(__file__).parent.parent.parent
+        script_path = os.path.join(current_dir, "scripts", "bash", "mg_external.sh")
+        run_cmd(
+            f"sh {script_path} {os.path.join(path, 'ceph_external')} {kubeconfig_path}",
+            timeout=100,
```

Comment:
In the shell script, there is used _timeout_ 120 for the individual commands, so I think the overall timeout for the whole script should be higher.

Reply:
done